files/en-us/web/javascript/reference/global_objects/date/tostring/index.html

```diff
@@ -26,7 +26,7 @@ <h3 id="Return_value">Return value</h3>
 <h2 id="Description">Description</h2>
 
 <p>{{jsxref("Date")}} instances inherit their <code>toString()</code> method from
-  {{jsxref("Date.prototype")}}, not {{jsxref("Object.prototype")}}.
+  {{jsxref("Date")}}, not {{jsxref("Object")}}.
```

Comment:
```suggestion
  {{jsxref("Date", "Date.prototype")}}, not {{jsxref("Object", "Object.prototype")}}.
```

I think it will read weirdly if the text content for these links says "Thing" rather than "Thing.prototype".